query_select.go

```diff
@@ -920,6 +920,17 @@ func (q *SelectQuery) whereExists(ctx context.Context) (bool, error) {
 	return n == 1, nil
 }
 
+func (q *SelectQuery) String() (string, error) {
```

Comment:
Let's just return `string` so we satisfy `Stringer` interface. We can panic on errors.